runtime/js.rs

```diff
@@ -13,10 +13,3 @@ pub fn deno_isolate_init() -> Snapshot {
   debug!("Deno isolate init with snapshots.");
   Snapshot::Static(RUNTIME_SNAPSHOT)
 }
-
-#[cfg(not(feature = "include_js_files_for_snapshotting"))]
-pub static SOURCE_CODE_FOR_99_MAIN_JS: &str = include_str!("js/99_main.js");
-
-#[cfg(feature = "include_js_files_for_snapshotting")]
-pub static PATH_FOR_99_MAIN_JS: &str =
-  concat!(env!("CARGO_MANIFEST_DIR"), "/js/99_main.js");
```

Comment:
Why are these getting removed? I think it's better to be explicit here than have embedders know which file to include

Reply:
It was necessary in the previous approach but I think I can restore this part